.claude/skills/screenshot-ui/SKILL.md

```
@@ -0,0 +1,200 @@
+
+## When to Use
+
+- Before creating a PR with UI changes
+- When the user asks to capture or preview UI changes visually
+- As Step 4.5 of the `create-pr` skill
+
+## Prerequisites
+
+1. **Playwright MCP available** — configured in `.cursor/mcp.json` and `.mcp.json` via `scripts/launch-playwright-mcp.sh`. Verify by calling `browser_snapshot`; if it errors, stop. If running using Cursor, the user also has to have it enabled in Cursor Settings.
+2. **Dev server running** on `https://localhost:8080` (`yarn start`). Check:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)